src/dotnet/commands/dotnet-build/CompileContext.cs

```diff
@@ -437,12 +437,7 @@ public CompilerIO(List<string> inputs, List<string> outputs)
         // computes all the inputs and outputs that would be used in the compilation of a project
         // ensures that all paths are files
         // ensures no missing inputs
-        public static CompilerIO GetCompileIO(
-            ProjectContext project,
-            string buildConfiguration,
-            string buildBasePath,
-            string outputPath,
-            ProjectDependenciesFacade dependencies)
+        public static CompilerIO GetCompileIO(ProjectContext project, string buildConfiguration, string buildBasePath, string outputPath, ProjectDependenciesFacade dependencies, bool isRootProject)
```

Comment:
Please, break the arguments in this line. I don't know about you, but I don't have that high of a resolution horizontal scrolling sucks when reading code.